### ISSUE TITLE
Windows 8.1 Compatibility

### ISSUE THREAD

**Issue by toqitahamid** [philferriere/dlwin#11]
Can I follow the instruction on windows 8.1 too? Will it be compatible to windows 8.1?

**Comment by philferriere** [comment]
I, personally, haven't tried. If you give it a shot, please let us know how it turns out.

**Comment by deltaz0** [comment]
Windows 7 compatibility; Posting this here in case it helps someone.

I followed this guide exactly for Windows 7 and it did not work at first (cuda installed but GPU not available error).

I then installed Visual Studio Community 2013 in addition to the 2015 version, and changed the line in my Path from

C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin

to

C:\Program Files (x86)\Microsoft Visual Studio 12.0\VC\bin

And now it works.
